search/public/index.html

```diff
@@ -33,6 +33,10 @@
 
       gtag('config', 'G-VLCP80LDQ7');
     </script>
+    <script
+      type="text/javascript"
+      src="https://maps.google.com/maps/api/js?key={}&v=3.31&v=weekly&libraries=places"
```

Comment:
Could this be loaded only in the Explore page?

Is there meant to be a key in there? We need to make it easy to configure this key on docker build.

Reply:
To only loaded in the explore page, I've tried with [this](https://github.com/leozdgao/react-async-script-loader) but the point to utilise google map isn't explicit in our case, because it's mainly just Reactivesearch take the mapkey as a parameter. Our call function is in the render(). [another reference](https://stackoverflow.com/questions/41709765/how-to-load-the-google-maps-api-script-in-my-react-app-only-when-it-is-require)

Reply:
Okay